src/applications/personalization/profile-2/components/direct-deposit/DirectDepositContent.jsx

```diff
@@ -0,0 +1,311 @@
+import { connect } from 'react-redux';
+
+import ReactCSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import EbenefitsLink from 'platform/site-wide/ebenefits/containers/EbenefitsLink';
+import { isLOA3, isMultifactorEnabled } from 'platform/user/selectors';
+
+import {
+  editModalToggled,
+  savePaymentInformation as savePaymentInformationAction,
+} from 'applications/personalization/profile360/actions/paymentInformation';
+import {
+  directDepositAccountInformation,
+  directDepositInformation,
+  directDepositIsSetUp,
+  directDepositUiState as directDepositUiStateSelector,
+} from 'applications/personalization/profile360/selectors';
+
+import BankInfoForm from 'applications/personalization/profile360/components/BankInfoForm';
+import PaymentInformationEditError from 'applications/personalization/profile360/components/PaymentInformationEditModalError';
+
+import ProfileInfoTable from '../ProfileInfoTable';
+import FraudVictimAlert from './FraudVictimAlert';
+import AdditionalInformation from './DirectDepositInformation';
+
+import { prefixUtilityClasses } from '../../helpers';
+
+// TODO: move this to a common location (in platform?) for easy sharing
+/**
+ * While not included with React, this hook is described in the React Hooks
+ * documentation:
+ * https://reactjs.org/docs/hooks-faq.html#how-to-get-the-previous-props-or-state
+ *
+ * as well as on usehooks.com:
+ * https://usehooks.com/usePrevious/
+ *
+ * This post does a deeper dive for those who are new to hooks:
+ * https://blog.logrocket.com/how-to-get-previous-props-state-with-react-hooks/
+ */
+const usePrevious = value => {
+  const valueRef = useRef();
+  useEffect(
+    () => {
+      valueRef.current = value;
+    },
+    // only run this useEffect hook if `value` changes
+    [value],
+  );
+  // this returns _before_ the `useEffect` call completes, so this will return
+  // whatever was stored in `ref.current` _before_ this function's `useEffect`
+  // callback runs
+  return valueRef.current;
+};
+
+export const DirectDepositContent = ({
+  isAuthorized,
+  isDirectDepositSetUp,
+  directDepositAccountInfo,
+  directDepositUiState,
+  saveBankInformation,
+  toggleEditState,
+}) => {
+  const editBankInfoButton = useRef();
+  const [formData, setFormData] = useState({});
+  const [showSaveSucceededAlert, setShowSaveSucceededAlert] = useState(false);
+  const previousUiState = usePrevious(directDepositUiState);
+
+  const isEditingBankInfo = directDepositUiState.isEditing;
+
+  // when we enter and exit edit mode...
+  useEffect(
+    () => {
+      // reset the bank info form when we enter edit mode
+      if (isEditingBankInfo) {
+        setFormData({});
+      }
+      // focus the edit button when we exit edit mode
+      if (isEditingBankInfo === false) {
```

Comment:
After successfully saving new bank information, the focus is still on the edit button. Is this desired, or should it go to the `We’ve saved your direct deposit information.` banner?

Reply:
I'm not sure yet. Gotta try this with VoiceOver turned on to see how it works. Related: I guess I need to add a better aria label on the Edit button regardless.